lib/ansible/modules/net_tools/dnsmadeeasy.py

```diff
@@ -368,12 +375,17 @@
 
 class DME2(object):
 
-    def __init__(self, apikey, secret, domain, module):
+    def __init__(self, apikey, secret, domain, sandbox, module):
         self.module = module
 
         self.api = apikey
         self.secret = secret
-        self.baseurl = 'https://api.dnsmadeeasy.com/V2.0/'
+
+        if sandbox:
+            self.baseurl = 'https://api.sandbox.dnsmadeeasy.com/V2.0/'
```

Comment:
I would suggest to also warn a bit about so users won't "forget" they are using the sandbox url
~~~python
self.module.warn(warning="Sandbox is enabled. All actions are made against the URL %s" % self.baseurl)
~~~

Reply:
Thanks for the suggestion. I will include this verbatim.